build.sbt

```diff
@@ -11,6 +11,7 @@ lazy val gcsFileSystem = (project in file("filesystems/gcs"))
 
 lazy val database = (project in file("database"))
   .settings(databaseSettings:_*)
+  .dependsOn(core) // used for migration
```

Comment:
ToL: Going to split migration, and its dependencies such as liquibase, core, etc., out into a separate artifact just after this PR. Fine for now.